ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -30,6 +30,13 @@ def eigh(tensor, name=None):
     return ivy.eigh(tensor)
 
 
+@to_ivy_arrays_and_back
+def eigvals(tensor, tol=None):
+    x = tensor.eig()
```

Comment:
Can we use `ivy.eighvals` as we have the function implemented on the functional API

Reply:
Hi, I believe that we can do that now. The reason I didn't do that is probably because when I worked on this function, only ivy.eig is available in the documentation. Should I change it?